### ISSUE TITLE
Sound will not play from within Git Bash, but it will play from within Windows CMD.  Why is that?

### ISSUE THREAD

**Issue by cagross** [git-for-windows/git#2788]
- [ X ] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

2.27.0 64 bit

```
$ git --version --build-options

git version 2.27.0.windows.1
cpu: x86_64
built from commit: 907ab1011dce9112700498e034b974ba60f8b407
sizeof-long: 4
sizeof-size_t: 8

```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?
Win 10
```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.18362.900]
(c) 2019 Microsoft Corporation. All rights reserved.

```

 - What options did you set as part of the installation? Or did you choose the
   defaults?
I can't remember :-/
```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

cat /etc/install-options.txt
Editor Option: VisualStudioCode
Custom Editor Path:
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled

```
 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

No.

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Bash

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
I created a simple JavaScript file (`sound.js`) with two lines:

console.log('Test message');
console.log("\007");

I then opened Git Bash for Windows and executed this line:  node sound.js

```
 - What did you expect to occur after running these commands?

I expected the message to be printed to the terminal, and a sound to be played.

 - What actually happened instead?

The message was printed to terminal, but no sound was played.

Note:

* If I open Windows CMD and execute the same command, both the message is printed to terminal and the sound is played.
* From the Git Bash command line, if I execute `echo -en "\007"` then the sound is played.

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

N/A

**Comment by dscho** [comment]
Git Bash actually runs in a MinTTY, and as far as I remember, the bell is turned off by default. You can turn it on via `~/.minttyrc`, there is even a GUI option for it (right-click on the window icon in the upper left of the Git Bash window and select `Options`). See https://github.com/mintty/mintty/issues/369 for the history of that feature.

**Comment by cagross** [comment]
Sorry, I should have mentioned two things :-/

1. In the MinTTY options, Terminal-->Bell is already set to 'Default Beep.'  And if I click the Play button next to it, the sound plays ([screenshot](https://i.imgur.com/rvLVUqt.jpeg)).

2.  From the command line, if I execute the command `echo -en "\007"` the sound plays as expected.  

Thoughts?  Is that enough to re-open this issue here?  Or do you still think it's best to follow up with MinTTY first?

Thanks.

**Comment by dscho** [comment]
Could you check whether the same thing occurs in a regular [MSYS2](https://msys2.github.io/) and/or in a regular [Cygwin](https://cygwin.com) (you will have to launch `mintty.exe` manually in the latter)?

**Comment by rimrul** [comment]
Could you also check the output of `type -a node`? 
Does the output include `node is aliased to 'winpty node'` or similar? If yes, could you try `node.exe sound.js`? Else could you try `winpty node sound.js`?

**Comment by cagross** [comment]
@dscho I tried MSYS2.  `node sound.js` fails with `bash: node: command not found`.  But also, an `ls` doesn't return anything at all, even though there are files/folders in this directory.  Sorry, I don't have any prior experience with MSYS2 :-/

@rimrul 

To be clear, you want me to check these in Git Bash for Windows--correct?  If so, the output is

> Could you also check the output of type -a node?

```
node is aliased to `winpty node.exe'
node is /c/Program Files/nodejs/node
```

> Does the output include node is aliased to 'winpty node' or similar? 

Yes.

> If yes, could you try node.exe sound.js? 

I tried it, and it works--the sound plays as expected.  Very nice--thanks for that.

> Else could you try winpty node sound.js?

I tried that, but the sound did not play.

So why exactly is `node.exe sound.js` printing the console message _and_ playing the sound, while `node sound.js` prints only the console message?  I might expect the latter to do neither.
